static/app/views/alerts/rules/metric/triggers/chart/thresholdsChart.tsx

```diff
@@ -28,6 +30,7 @@ import type {MetricRule, Trigger} from '../../types';
 import {AlertRuleThresholdType, AlertRuleTriggerType} from '../../types';
 
 type DefaultProps = {
+  anomalies: Anomaly[];
```

Comment:
might be nice to create a context just for anomalies so we can avoid prop drilling

Reply:
See https://github.com/getsentry/sentry/pull/78238#discussion_r1793954292